test/node/node_test.cpp

```diff
@@ -155,19 +163,21 @@ TEST(NodeTest, SimpleSubkeys) {
 }
 
 TEST(NodeTest, StdArray) {
-  std::array<int, 5> evens;
-  evens[0] = 2;
-  evens[1] = 4;
-  evens[2] = 6;
-  evens[3] = 8;
-  evens[4] = 10;
-
+  std::array<int, 5> evens {{ 2, 4, 6, 8, 10 }};
   Node node;
   node["evens"] = evens;
-  std::array<int, 5> actualEvens = node["evens"].as<std::array<int, 5> >();
+  std::array<int, 5> actualEvens = node["evens"].as<std::array<int, 5>>();
   EXPECT_EQ(evens, actualEvens);
 }
 
+TEST(NodeTest, StdArrayWrongSize) {
+  std::array<int, 3> evens {{ 2, 4, 6 }};
+  Node node;
+  node["evens"] = evens;
+  EXPECT_THROW_REPRESENTATION_EXCEPTION((node["evens"].as<std::array<int, 5>>()),
```

Comment:
I'd prefer inlining this rather than making a separate macro, since you're only using it once.